dev/fattest.databases/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+#TODO once we upgrade fattest.simplicty to java 8 this package could be moved there.
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: FAT Database infrastructure
+Bundle-SymbolicName: fattest.databases
+Bundle-Description: FAT infrastructure; version=${bVersion}
+
+Export-Package: \
+	componenttest.topology.database.container;version=1.0.16
+	
+Include-Resource: \
+  META-INF=resources/META-INF
+
+test.project: true
+
+-buildpath: \
+	com.ibm.ws.componenttest:public.api;version=1.0.0,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	org.testcontainers:database-commons;version=1.12.3,\
+	org.testcontainers:jdbc;version=1.12.3,\
```

Comment:
I removed the compile time dependencies on the database specific components by using reflection to create the containers.